code/game/machinery/cell_charger.dm

```diff
@@ -136,10 +136,11 @@
 	if(charging.percent() >= 100)
 		return
 
-	var/main_draw = use_power_from_net(charge_rate * seconds_per_tick, take_any = TRUE) //Pulls directly from the Powernet to dump into the cell
+	var/main_draw = charge_rate * seconds_per_tick
 	if(!main_draw)
 		return
-	charging.give(main_draw)
-	use_power(charge_rate / 100) //use a small bit for the charger itself, but power usage scales up with the part tier
+
+	use_energy(main_draw * 0.01) //use a small bit for the charger itself, but power usage scales up with the part tier
```

Comment:
```suggestion
	use_energy(main_draw * GLOB.CHARGELEVEL) //use a small bit for the charger itself, but power usage scales up with the part tier
```

Since they have the same value. Or maybe the meaning here is different? idk you decide

Reply:
That's just extra power usage with heat inefficiency or something.